### DISCUSSION TITLE
Creating workflow with local subjects

### DISCUSSION THREAD

**Discussion by aliplump** [bonsai-rx/bonsai#1664] [Q&A]
Hi! I am pretty new to using Bonsai and have a fairly simple question regarding the scope of subjects. How can I create a GroupWorkflow where all the subjects within it are defined at a local scope only? i.e. the workflow would have a solid border as mentioned in this: https://bonsai-rx.org/docs/articles/subjects.html#scope-of-subjects

**Comment by bruno-f-cruz** [comment]
Hi,
Without knowing the specifics of your use-case, I would default to the `Defer` operator.

**Reply by jbscherrer** [reply]
Is it possible to make the solid-border GroupWorkflow in this screenshot from the docs? Or is that image for illustration purposes only?
![image](https://github.com/bonsai-rx/bonsai/assets/39060975/a935dfe4-79f8-4766-b4e1-d2690517c2bb)

**Reply by bruno-f-cruz** [reply]
Wouldn't that essentially be a Defer node? Sorry not sure I follow what you are exactly trying to achieve.

**Reply by jbscherrer** [reply]
No you're absolutely right, it sounds like we should just use `Defer` nodes. That figure from the docs had us believe it was possible to create a GroupWorkflow with local scope, and we were getting confused trying to figure out how to create one.
